Tasks/PublishCodeCoverageResultsV1/publishcodecoverageresults.ts

```diff
@@ -54,12 +54,16 @@ async function run() {
 }
 
 // Resolves the specified path to a single item based on whether it contains wildcards
-function resolvePathToSingleItem(workingDirectory:string, pathInput: string) : string {
+function resolvePathToSingleItem(workingDirectory:string, pathInput: string, isDirectory: boolean) : string {
     // Default to using the specific pathInput value
     var resolvedPath: string = pathInput;
 
-    // Does the pathInput value contain wildcards?
     if (pathInput) {
+
+        // Find match patterns won't work if the directory has a trailing slash
+        if (isDirectory && (pathInput.endsWith("/") || pathInput.endsWith("\\"))) {
```

Comment:
This will not work if users use multiple slashes (such as `\\code-coverage\\`. However, the task does not require the input to escape any of the slashes, so I only check for a single slash.